src/mds/FSMap.cc

```diff
@@ -1090,10 +1090,12 @@ std::vector<mds_gid_t> FSMap::stop(mds_gid_t who)
     if (other_info.rank == info.rank
         && other_info.state == MDSMap::STATE_STANDBY_REPLAY) {
       standbys.push_back(other_gid);
-      erase(other_gid, 0);
     }
   }
 
+  for (const auto other_gids : standbys)
+    erase(other_gids, 0);
+
```

Comment:
@aliakseimakarau this compilation error is causing the "make check" test to fail:
```
../src/mds/FSMap.cc:1096:19: error: loop variable 'other_gids' creates a copy from type 'const mds_gid_t' [-Werror,-Wrange-loop-construct]
  for (const auto other_gids : standbys)
                  ^
../src/mds/FSMap.cc:1096:8: note: use reference type 'const mds_gid_t &' to prevent copying
  for (const auto other_gids : standbys)
       ^~~~~~~~~~~~~~~~~~~~~~~
                  &
1 error generated.
```

Reply:
... and minor nit: needs enclosing `{}`.

Reply:
Hi Laura @ljflores , hi Venky @vshankar ,

i see the tests have passed, could you please tell me if i have to do something further? Thank you very much!

Reply:
I'll include this in my test branch. Once teuthology tests pass, this will be merged.